cardano-db-sync/src/Cardano/DbSync/Util.hs

```diff
@@ -68,6 +74,11 @@ import qualified Ouroboros.Network.Point as Point
 import           Ouroboros.Network.Block (Point (..), SlotNo (..))
 
 
+
+-- | Currently we use Bitcoin alphabet for representing addresses in base58
+addrToBase58 :: Crypto crypto => Addr crypto -> ByteString
+addrToBase58 = encodeBase58 bitcoinAlphabet . serialize'
```

Comment:
That will not be correct.Mu understanding is that we will be using bech32.

Reply:
For the pool hash? After talking with Matthias, we arrived at the following conclusion.
We use base64 encoding here.
Submissions are identified by the subject's Bech32-encoded Ed25519 public key (all lowercase).
An Ed25519 public key is a 64-byte string. We'll typically show such string in base16.
base64 is fine too, more concise. But bech32 is definitely overkill here.
I used base58 since that is the same format Byron used and can be used without changing the database format.